src/main/kotlin/se/zensum/ktorSentryFeature/ReadHostname.kt

```diff
@@ -0,0 +1,28 @@
+package se.zensum.ktorSentry
```

Comment:
Is there a reason this is in another file? It's not really shared behaviour as its a single purporse library.

Reply:
Well it is not shared behavior, but we might want to separate our code a bit anyway? This library could remain a single purpose library with only the Sentry feature publicly exposed, yet have 10 000 lines of code (it would of course be bit strange for this lib), should it still have been in just one file? When in doubt, I prefer too create a separate file rather than keeping more code stuffed in one. After all, getting a host name is not really a core feature of this library so I felt it was better in a file of its own.